mobile/lib/pages/backup/drift_backup.page.dart

```diff
@@ -161,6 +165,27 @@ class _DriftBackupPageState extends ConsumerState<DriftBackupPage> {
                       ),
                     ),
                   },
+                  FutureBuilder(
+                    future: Permission.notification.isGranted,
+                    builder: (context, snapshot) {
+                      final isBackupEnabled = ref
+                          .watch(appSettingsServiceProvider)
+                          .getSetting(AppSettingsEnum.enableBackup);
+
+                      if (snapshot.hasData && !snapshot.data! && Platform.isAndroid && isBackupEnabled) {
```

Comment:
We can probably extract the data conditions to a separate variable to make things easier to read
```suggestion
                      if (snapshot.hasData && !snapshot.data! && CurrentPlatform.isAndroid && isBackupEnabled) {
```